README.md

```diff
@@ -558,7 +558,7 @@ If you need an app for **menstrual cycle tracking** please don't use any apps li
 - [ProExpense](https://github.com/arduia/ProExpense/) - A simple free finance note to safely record daily expenses.
 - [IHateMoney](https://ihatemoney.org/) - Manage your shared expenses, easily.
 - [Nextcloud Cospend](https://apps.nextcloud.com/apps/cospend) - A group/shared budget manager inspired by the great IHateMoney.
-   - [MoneyBuster](https://gitlab.com/eneiluj/moneybuster/) - Android client for Nextcloud Cospend and [IHateMoney](https://ihatemoney.org/) servers.
+   - [MoneyBuster](https://gitlab.com/eneiluj/moneybuster/) - Android client for Nextcloud, Cospend and [IHateMoney](https://ihatemoney.org/) servers.
```

Comment:
I think it is Nextcloud Cospend:
* https://github.com/eneiluj/cospend-nc
* https://apps.nextcloud.com/apps/cospend

Reply:
Fixed, thanks!